decidim-core/app/controllers/decidim/application_controller.rb

```diff
@@ -6,11 +6,22 @@ class ApplicationController < ActionController::Base
     include Decidim::LocaleSwitcher
     include NeedsAuthorization
     protect_from_forgery with: :exception, prepend: true
+    helper_method :current_participatory_process
 
     layout "layouts/decidim/application"
 
     def user_not_authorized_path
       decidim.root_path
     end
+
+    private
+
+    def current_participatory_process
```

Comment:
I'm not sure tbh, @josepjaume added this method here: https://github.com/AjuntamentdeBarcelona/decidim/commit/dcc176450105e6727cb702f5ae9fbc9001c47c3b 

Maybe he can give some insight.